packages/generator-langium/templates/packages/extension/package.json

```diff
@@ -24,17 +27,21 @@
     ],
     "main": "./out/extension/main.cjs",
     "scripts": {
+        "clean": "shx rm -fr *.tsbuildinfo out syntaxes",
         "vscode:prepublish": "npm run build && npm run lint",
```

Comment:
There is no `lint` script in this package.

Reply:
Thank you, I used `npm run --prefix ../.. lint` ⬇️